fix: button favorite and 3 dots menu in activity detail have no accessible name - EXO-62416 .

akhanfir opened the pull request:
Before this change, after creating an activity in a space when you open DevTools of your browser and in the flagship tab, run the accessibility check, the favorite button and the 3 dots menu button have no accessible name. To resolve this problem, in the News_eu.properties file, added the labels following find it easily when searching, Remove from favorites and Open activity menu to the Archive article, Unarchive article and menu 3 buttons points of the activity. After this change, these faulty elements are no longer displayed in the flagship tab, launch accessibility.

aycherif committed:
TASK-62301: Create FB maintenance and update projects versions/dependencies

sofyenne committed:
fix : enable to display published news images selected from existing upload for non members - EXO-65111 (#874)

This change is going to display the published news images selected from existing upload for non space members.

sofyenne committed:
fix : enable to display published news images selected from existing upload for non members - EXO-65111 (#875)

akhanfir committed:
fix: Article drafts edition makes the article displayed on top of the news app - EXO-56012. (#950)

Before this fix, when post several articles in a space and in news app, click on three dots>edit button for the last posted article then type some inputs then click on cancel, this article is displayed on the top of the posted articles list. in fact, it is that the draft which was created and the article existing in the same property of the date modified (dateModified) to fix this problem, during creation of draft in set another property (lastModifiedDate) and during get all the articles by modifying the tree property according to the filter of this list if it is drafted by using it as a property lastModifiedDate if not by using the same existing property. After this change, the routed list is sorted either by lastModifiedDate if the filter is the draft article list or by dateModified if the list has another filter.

akhanfir committed:
fix: button favorite and 3 dots menu in activity detail have no accessible name - EXO-62416 .
Before this change, after creating an activity in a space when you open DevTools of your browser and in the flagship tab, run the accessibility check, the favorite button and the 3 dots menu button have no accessible name. To resolve this problem, in the News_eu.properties file, added the labels following find it easily when searching, Remove from favorites and Open activity menu to the Archive article, Unarchive article and menu 3 buttons points of the activity. After this change, these faulty elements are no longer displayed in the flagship tab, launch accessibility.

rdenarie commented on services/src/main/resources/locale/portlet/news/News_en.properties:
```diff
@@ -88,15 +88,15 @@ news.activity.clickToShowDetail=Click here for more news details
 news.archive.btn.confirm=Confirm
 news.archive.btn.cancel=Cancel
-news.archive.action=Archive article
+news.archive.action=Bookmark it to easily find it when searching
```
Why this value is changed ?

akhanfir replied:
This was requested in the task, but if you wish, we can keep the old text.

rdenarie replied:
can you confirm that the key news.archive.action correspond to "Favorite" action ? 
It seems related to different things

rdenarie replied:
In addition, labels like that must be changed in crowdin

akhanfir replied:
As you said preferably that we add another key to avoid changing another label so it must that I add another key and I would change the description of the PR.

rdenarie commented on services/src/main/resources/locale/portlet/news/News_en.properties:
```diff
@@ -88,15 +88,15 @@ news.activity.clickToShowDetail=Click here for more news details
 news.archive.text=Sorry, you don't have access to this article. Please contact your administrator.
 
-news.unarchive.action=Unarchive article
+news.unarchive.action=Remove from favorites
```
Why this value is changed ?

akhanfir replied:
As you said preferably that we add another key to avoid changing another label so it must that I add another key and I would change the description of the PR.

akhanfir committed:
fix: News Card news template does not have a unique ID - EXO-62572 . (#976)

Before this change, when you add 2 news stories and post them to a news list on the snapshot page and configure the news list on the snapshot page to use the news card template, in search source code id=space-link. There are 2 tags with this ID. To resolve this issue, update id=space-link in newsCardsViewItem to id= space-link-(item.activityId). After this change, in the News Card template space link in each card to their ID.

Jihed525 committed:
Merge branch 'feature/maintenance' into TASK-62416

ahamdi committed:
Merge branch 'feature/maintenance' into TASK-62416
